src/interfaces/BoundingBoxInterface.java

```diff
@@ -0,0 +1,86 @@
+package interfaces;
+
+import main.BoundingBox;
+
+/**
+ * BoundingBoxes (axis-aligned) are used to calculate collisions between fish.
+ * Each fish is represented in the 2-D plane of the screen with a BoundingBox.
+ * 
+ * @author Michiel
+ */
+public interface BoundingBoxInterface {
+
+	  /**
```

Comment:
each of these lines all gives me checkstyle warnings:
"Line contains a tab space"
Does someone wants to fix it or I can do it as well?
< review, layout